lib/workable/client.rb

```diff
@@ -257,6 +257,19 @@ def event(id)
       @transform_to.apply(:event, get_request("events/#{id}"))
     end
 
+    def refresh_access_token(client_id, client_secret, refresh_token)
+      uri = URI.parse('https://www.workable.com/oauth/token')
```

Comment:
Can you move this to an EnvVar or a constant?

Reply:
I think we can use `post_request` method available.

Reply:
The post_request is expecting the default headers as well as the api_url.
Not sure if we want to refactor that method, which will diverge from the forked repository. I will move the URL to a constant. Thanks for the review!

Reply:
https://github.com/ScreenLoop/workable-rb/blob/e74b57ac7c4b43ab5614ef3a97397d0fdfda8d70/lib/workable/client.rb#L299

`post_request` accepts a block and you can change the headers there

Reply:
Not sure if I get your point.
In that case, the do_request is called, the block I can change is this:
`yield(request) if block_given?`

Reply:
No need to change after discussing this IRL